examples/using-wordpress/gatsby-config.js

```diff
@@ -23,10 +23,43 @@ module.exports = {
         // If false, then the asumption is made that the site is self hosted.
         // If true, then the plugin will source its content on wordpress.com using the JSON REST API V2.
         // If your site is hosted on wordpress.org, then set this to false.
-        hostingWPCOM: false,
+        hostingWPCOM: true,
         // If useACF is true, then the source plugin will try to import the Wordpress ACF Plugin contents.
-        // This feature is untested for sites hosted on Wordpress.com
+        // This feature is untested for sites hosted on Wordpress.com.
+        // Defaults to true.
         useACF: true,
+        auth: {
+          // If auth.user and auth.pass are filled, then the source plugin will be allowed
+          // to access endpoints that are protected with .htaccess.
+          htaccess_user: "your-htaccess-username",
+          htaccess_pass: "your-htaccess-password",
+          htaccess_sendImmediately: false,
+
+          // If hostingWPCOM is true then you will need to communicate with wordpress.com API
+          // in order to do that you need to create an app (of type Web) at https://developer.wordpress.com/apps/
+          // then add your clientId, clientSecret, username, and password here
+          wpcom_app_clientSecret:
+            "NMPnXYFtj2gKas7V1kZyMxr7oLry9V5ZxIyBQGu2txjVHg0GhFz6RYcKopkHICYg",
+          wpcom_app_clientId: "54793",
+          wpcom_user: "gatsbyjswpexample@gmail.com",
+          wpcom_pass: "very-secured-password",
+        },
+        // Set verboseOutput to true to display a verbose output on `npm run develop` or `npm run build`
+        // It can help you debug specific API Endpoints problems
+        verboseOutput: false,
+        // Set how many pages are retrieved per API request
+        perPage: 100,
+        // Search and Replace Urls across WordPress content
+        searchAndReplaceContentUrls: {
+          sourceUrl: "https://source-url.com",
+          replacementUrl: "https://replacement-url.com",
+        },
+        // Exclude specific endpoint manufacturers
+        // Example: [`yoast`] will exclude all `/yoast/` endpoints such as `/yoast/v1/statistics` from fetch. 
+        // excludedManufacturers: [`disqus`, `yoast`],
+        // Exclude sepcific endpoint types
+        // Example: [`comments`] will exclude all `comments` type endpoints such as `/wp/v2/comments` from fetch. 
+        // excludedTypes: [`comments`, `media`],
```

Comment:
Please don't change this config with example data - `README.md` is place for it - this is config for demo site - https://using-wordpress.gatsbyjs.org/ (I know that it currently doesn't build)

Reply:
Got it I'll remove.